packages/clarity-cli/package.json

```diff
@@ -22,12 +25,15 @@
     "@types/fs-extra": "^7.0.0",
     "@types/mocha": "^5.2.6",
     "@types/node": "^10",
+    "@types/node-fetch": "^2.3.4",
+    "@types/tar": "^4.0.0",
     "chai": "^4.2.0",
     "globby": "^9.2.0",
     "mocha": "^6.1.4",
-    "nyc": "^14.1.1",
```

Comment:
What's the issue with `14.1.1`?

Reply:
When nyc-v14.x first came it, it wasn't able to generate code coverage reports correctly for latest stable version of typescript. I haven't tested it out since then though.

On that note - we should get the typescript+mocha+nyc config setup and standardized across the packages, and then get CircleCI + codecov.io integration configured.